views/includes/global-header-next.njk

```diff
@@ -150,3 +150,60 @@
         </div>
     </div>
 </header>
+
+<script>
```

Comment:
Slightly controversial but this is the quickest way to add event listeners for these elements, by triggering the script here the part of the DOM we care about is ready and it means users can click the navigation links before the rest of the page has finished resulting in a snappy time to interactive for common navigation tasks. 💭 thought's welcome on if there is a nicer way to approach this.